tests/rules/no-unused-expressions.js

```diff
@@ -74,12 +74,13 @@ ruleTester.run("no-unused-expressions", rule, {
         },
 
         // Babel-specific test cases.
-      "let a = do { if (foo) { foo.bar } }",
-      "let a = do { foo }",
-      "let a = do { let b = 2; foo; }",
-      "let a = do { (foo + 1) }",
-      "let a = do { if (foo) { if (foo.bar) { foo.bar } } }",
-      "let a = do { if (foo) { if (foo.bar) { foo.bar } else if (foo.baz) { foo.baz } } }",
+        "let a = do { if (foo) { foo.bar } }",
+        "let a = do { foo }",
+        "let a = do { let b = 2; foo; }",
+        "let a = do { (foo + 1) }",
+        "let a = do { if (foo) { if (foo.bar) { foo.bar } } }",
+        "let a = do { if (foo) { if (foo.bar) { foo.bar } else if (foo.baz) { foo.baz } } }",
+        "foo.bar?.();",
```

Comment:
@kaicataldo these lines had wrong indentation, so I also fixed that along the way. I also noticed that the code uses semi-colons inconsistently, should probably be addressed for readability?

Reply:
@lehni re: semicolons, I don't think it's a big deal, but fine if we make the babel-specific ones consistent. The others are just copied from ESLint directly.

Reply:
@existentialism ok, shall I do so in this PR?

Reply:
(the others don't have the same semi-colon style, btw)